source/release-notes.txt

```diff
@@ -10,6 +10,48 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.20
+--------------
+
+*Released November, 2019*
+
+- New and improved :guilabel:`New Connection` user experience with
+  support for all connection options.
+
+- Added support for:
+
+  - Cloning :guilabel:`Favorites`.
+
+  - Adding, viewing and editing documents in JSON.
```

Comment:
Technically it was always JSON, just not a super convenient way of doing so. I think we could say something like "Added new JSON mode for managing documents." An important thing to highlight here is that you can now insert multiple documents at once (common user request). So maybe append a sentence like "With JSON mode, you can now insert multiple documents at once."